public/pages/MonitorDetails/containers/Triggers/Triggers.js

```diff
@@ -24,12 +25,35 @@ export function getUnwrappedTriggers(monitor) {
         return trigger[TRIGGER_TYPE.BUCKET_LEVEL];
       case MONITOR_TYPE.DOC_LEVEL:
         return trigger[TRIGGER_TYPE.DOC_LEVEL];
+      case MONITOR_TYPE.COMPOSITE_LEVEL:
+        return trigger[TRIGGER_TYPE.COMPOSITE_LEVEL];
       default:
         return trigger[TRIGGER_TYPE.QUERY_LEVEL];
     }
   });
 }
 
+const expressionsToCondition = (expressions) => {
+  const conditionMap = {
```

Comment:
Non-blocking nitpick: I think I've seen this map in a few places. Could be worthwhile to extract it into a helper file somewhere.